interface1.js

```diff
@@ -884,35 +884,200 @@ window.addEventListener('load', function(){
+								}
+							}
+						}else{
+							if (Number(spotForm.genStringsMin.value) <= 0){
+								minProblem = 1;
+							}
+							if (Number(spotForm.genStringsMax.value) <= 0){
+								maxProblem = 1;
+							}
+						}
+					}
+					if (minProblem === 0 && maxProblem === 0){
+						/* checking if the input value is more than 10*/
+						for(var i=0; i<length; i++){
+							if (length > 1){
+								if (spotForm.genStringsInput[i].value !== ""){
+									if (Number(spotForm.genStringsMin[i].value) >= 10){
+										minProblem = 1;
+									}
+									if (Number(spotForm.genStringsMax[i].value) >= 10){
+										maxProblem = 1;
+									}
+								}
+							}else{
+								if (Number(spotForm.genStringsMin.value) >= 10){
+									minProblem = 1;
+								}
+								if (Number(spotForm.genStringsMax.value) >= 10){
+									maxProblem = 1;
+								}
+							}
+						}
+						var minmax = 0; //flag checker for min value being less than max value
+						if (minProblem === 0 && maxProblem === 0){
+							/* checking if the input is more than 5 and if min value is less than max value */
+							for(var i=0; i<length; i++){
+								if (length > 1){
+									if (spotForm.genStringsInput[i].value !== ""){
+										if (Number(spotForm.genStringsMin[i].value) >= 5){
+											minProblem = 1;
+										}
+										if (Number(spotForm.genStringsMax[i].value) >= 5){
+											maxProblem = 1;
+										}
+										if (Number(spotForm.genStringsMax[i].value) <  Number(spotForm.genStringsMin[i].value)){
+											minmax = 1;
+										}
+									}
+								}else{
+									if (Number(spotForm.genStringsMin.value) >= 5){
+										minProblem = 1;
+									}
+									if (Number(spotForm.genStringsMax.value) >= 5){
+										maxProblem = 1;
+									}
+									if (Number(spotForm.genStringsMax.value) <  Number(spotForm.genStringsMin.value)){
+										minmax = 1;
+									}
+								}
+							}
+							if (minmax === 0){
+								/* display warning if the input value is more than 5 */
+								if (minProblem === 1 || maxProblem === 1){
+									minProblem = 0;
+									maxProblem = 0;
+									displayWarning("Min or Max input is greater than or equal to 5 which may cause your browser to freeze due to too many terminal strings being generated.");
+								}
+								var length = spotForm.genStringsInput.length;
+								if(length === undefined) {
+									length = 1;
+								}
+								var inputList = spotForm.genStringsInput.value;
+								var min = spotForm.genStringsMin.value;
+								var max = spotForm.genStringsMax.value;
 
-			if(inputList !== "") {
-				inputList = inputList.trim().split(' ');
-				var currGenStringsList = generateTerminalStrings(inputList, min, max)
-				displayStringsTable(currGenStringsList);
 
-				if(genStringsList) {
-					genStringsList = genStringsList.concat(currGenStringsList);
-				}
-				else {
-					genStringsList = currGenStringsList;
+								for(var i=0; i<length; i++){
```

Comment:
Add a comment indicating that this is where the actual calculation of the terminal strings happens.

Reply:
I'm hoping that with the flipped logic I suggested above, this chunk would be in the "else" block (where no errors had been encountered), instead of being embedded in half a dozen nested if statements.